xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-templates/src/main/resources/templates/macros.vm

```diff
@@ -3212,3 +3212,35 @@ Recursive title display detected!##
 #macro (unwrapXPropertyDisplay $displayOutput)
 $!stringtool.removeEnd($stringtool.removeStart($displayOutput, '{{html clean="false" wiki="false"}}'), '{{/html}}')##
 #end
+
+#**
+ * XWiki Standard use only.
+ * $watchStatus is a WatchedEntityReference.WatchedStatus returned by script services 
+ * similar to $services.notification.watch.getLocationWatchedStatus
+ * Returns the FA4 class of the icon corresponding to the provided watchStatus in $watchIcon
+ * AND
+ * Returns the status label in $watchText
+ * @since 17.4.2
```

Comment:
In the PR description you said that this won't be backported, why do you indicate 17.4.2 then?